index.js

```diff
@@ -86,48 +105,48 @@ Compare.prototype = {
       : this._setPosition(this._getX(e));
   },
 
-  _onMouseUp: function() {
+  _onMouseUp: function () {
     document.removeEventListener('mousemove', this._onMove);
     document.removeEventListener('mouseup', this._onMouseUp);
     this.fire('slideend', { currentPosition: this.currentPosition });
   },
 
-  _onTouchEnd: function() {
+  _onTouchEnd: function () {
     document.removeEventListener('touchmove', this._onMove);
     document.removeEventListener('touchend', this._onTouchEnd);
   },
 
-  _getX: function(e) {
+  _getX: function (e) {
     e = e.touches ? e.touches[0] : e;
     var x = e.clientX - this._bounds.left;
     if (x < 0) x = 0;
     if (x > this._bounds.width) x = this._bounds.width;
     return x;
   },
 
-  _getY: function(e) {
+  _getY: function (e) {
     e = e.touches ? e.touches[0] : e;
     var y = e.clientY - this._bounds.top;
     if (y < 0) y = 0;
     if (y > this._bounds.height) y = this._bounds.height;
     return y;
   },
 
-  setSlider: function(x) {
+  setSlider: function (x) {
     this._setPosition(x);
   },
 
-  on: function(type, fn) {
+  on: function (type, fn) {
```

Comment:
Why is this PR introducing spaces before parens?

Reply:
I think it was automatically applied by my editor as a mistake. If it's irksome I can remove them?

Reply:
👍 I think this should PR shouldn't make formatting decisions stay focused on its intent.

Reply:
Is it okay if I change one of the vars to a const

Reply:
changing a var to a const isn't a formating decision. It's the right thing to do 👿